### ISSUE TITLE
[Bug] GotoFile failed finding file

### ISSUE THREAD

**Issue by Kristinita** [sagold/FuzzyFilePath#39]
### Summary

I can not open files, I referenced in my HTML-documents.
### Expected behavior

Successful open files.
### Actual behavior

I included `log: true,` in my `sublime-settings` file. When I run `ffp_goto_file` command, I get output in console — `FFP  GotoFile failed finding file $Path to my file`. Example:

```
FFP  GotoFile failed finding file E:\wamp\www\articles_css\style_right.css
```
### Steps to reproduce

Example of my code: https://github.com/Kristinita/Kristinita.github.io/blob/master/Sublime-Text/SashaSublime.html#L108

``` html
<!-- Иконочка Сашеньки -->
<link rel="shortcut icon" href="../other_images/Sasha.ico" type="image/x-icon">
<link rel="icon" href="../other_images/Sasha.ico" type="image/x-icon">
<!-- Стиль страницы -->
<link rel="stylesheet" href="../articles_css/style_right.css">
<link rel="stylesheet" type="text/css" href="../bower-components/rainbow/themes/css/espresso-libre.css">
```

I put cursor in any place between quotes between `href` → I run `ffp_goto_file` command → my files do not open, I get error in console.

All relative paths is correct. I also insert absolute paths, but I get same result.
### Environment

**Operating system and version:**
Windows 10.0.14393
**Sublime Text:**
Build 3126

Thanks.

**Comment by sagold** [comment]
Hi Kristinita.

Thanks for the detailed report. I am aware of this issue and hope to find some time soon. As a side note, for some path replacements (replace_on_insert) the filepath cannot be reconstructed.

**Comment by sagold** [comment]
I made some changes to goto_file which should resolve this issue. The changes are currently in develop for testing.

- querying a cleaned string in cache 7f46d628f90a0fbd38837a77252a1f2773755315
- independent of cache, check if the file is a valid path 2208c64b8b5d6bd9362fee9e214edd73917d218d

**Comment by sagold** [comment]
The patch has been released with v0.5.0.
